apis/v1/shared_types.go

```diff
@@ -25,7 +25,7 @@ import (
 // with "Core" support:
 //
 // * Gateway (Gateway conformance profile)
-// * Service (Mesh conformance profile, experimental, ClusterIP Services only)
+// * Service (Mesh conformance profile, ClusterIP Services only)
```

Comment:
Making this standard with `Service` is where we'll need buy in from

/cc @thockin